### ISSUE TITLE
The Lord of Celebration divination card shows no results (by default).

### ISSUE THREAD

**Issue by stormbringer766** [domialex/Sidekick#341]
By default when you look this up it has "Shaper" influence selected. It isn't, so you get no results.

```
Rarity: Divination Card
The Lord of Celebration
--------
Stack Size: 1/4
--------
Sceptre of Celebration
Shaper Item
--------
Though they were a pack of elite combatants, the Emperor's royal guards were not ready to face one of his notorious parties.
```

The regular expression probably need to be separated by type and section to avoid stuff like this.

**Comment by leMicin** [comment]
We detect the line Shaper Item as an influence in this case.

**Comment by stormbringer766** [comment]
@leMicin There are a few places this happens. One regular expression overrides another. I had a Vaal gem do something similar. The name parsed correctly but then one of the item regular expressions changed it. I can't find it now so I can't reproduce but I did watch it in debug.

**Comment by Blinke** [comment]
This is fixed in #343.
